src/server/db/queries.ts

```diff
@@ -26,13 +26,64 @@ export const queryJobResult = async (jobId: string): Promise<MinimalJobResultsIn
     return { ...results, resultsType: results.resultsPath ? 'APPROVED' : 'ENCRYPTED' } as MinimalJobResultsInfo
 }
 
-export const checkMemberAllowedStudyReview = async (studyId?: string, identifier = getOrgSlugFromActionContext()) => {
+export const checkUserAllowedJobView = async (jobId?: string, memberIdentifier?: string) => {
```

Comment:
the `checkUserAllowedJobView` and `checkUserAllowedStudyView` methods are seemingly getting a bit complex with a lot of logic around both optional arguments, im just thinking out loud but reading through it i could see how this might be confusing at first glance. i wonder if there's a way to split out these methods to be more granular with required args 🤔